lib/main_common.pm

```diff
@@ -1426,7 +1426,7 @@ sub load_x11tests {
         if (!is_krypton_argon && !is_kde_live) {
             loadtest "x11/amarok";
         }
-        loadtest "x11/kontact" unless is_kde_live;
+        loadtest "x11/kontact" if (is_x86_64 || is_aarch64 || is_riscv) && !(is_kde_live);
```

Comment:
IIUC this fails the CI test as now kontact would not be executed in a default kde scenario anymore at all which is not what the git commit message or the PR subject says and probably wrong

Reply:
@okurz @foursixnine I added more [VRs](https://openqa.opensuse.org/tests/overview?build=cedvid%2Fos-autoinst-distri-opensuse%2319297). 

kontact and khelpcenter are executed in kde scenario unless it's kde live and it's not loaded in kde unsupported archs or non-kde environment (gnome)

Reply:
then why is kontact missing from the "default schedule" as visible in https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/9077891561/job/24945718903?pr=19297#step:3:105 ?

Reply:
> then why is kontact missing from the "default schedule" as visible in https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/9077891561/job/24945718903?pr=19297#step:3:105 ?

found it: @cedvid is removing it from the default, it shows up because the expected output contains the modules.

So Cedric, `t/data/test_schedule.out` needs to have kcontact removed. In the meantime, create a note in a new file explaining what to do when the isotovideo check fails.

Reply:
Again, the subject says "Skip loading kontact and khelpcenter on unsupported architecture". Why should kontact be missing from the default schedule then? Or is the default scenario from a non-default architecture?

Reply:
https://progress.opensuse.org/issues/160439